components/camel-script/pom.xml

```diff
@@ -78,23 +78,14 @@
             <version>${jython-version}</version>
             <scope>test</scope>
         </dependency>
-        <!-- for camel-script groovy upgrade requires rework as camel-itest-karaf fails to succeed 
```

Comment:
not really. because karaf itests uses the specific version. Please see this line;
https://github.com/apache/camel/blob/master/platforms/karaf/features/src/main/resources/features.xml#L1943